templates/timeframe-withlocation.php

```diff
@@ -33,10 +35,10 @@
             ) {
 
                 // Merge overlapping timeframes
-				$ranges = \CommonsBooking\Helper\Helper::mergeRangesToBookableDates($data['ranges']);
+				$ranges = Helper::mergeRangesToBookableDates($data['ranges']);
 
 	            foreach ($ranges as $range) {
-		            echo commonsbooking_sanitizeHTML( \CommonsBooking\Model\Timeframe::formatBookableDate($range['start_date'], $range['end_date']) ) . '<br>';
+		            echo commonsbooking_sanitizeHTML( Timeframe::formatBookableDate($range['start_date'], $range['end_date']) ) . '<br>';
```

Comment:
Bei den Templates finde ich es ehrlich gesagt sinnvoller nicht das "use" Statement zu nutzen, sondern den ganzen Pfad zu nehmen. Vor allem, wenn nur eine Referenz existiert. Dadurch lässt sich direkt nachvollziehen, welche Klasse genau gemeint ist. Zum Beispiel gibt es Timeframe als Klasse in  Model, CustomPostType und Repository.

Reply:
Das sehe ich auch so.

Reply:
✅